docs/backend-system/core-services/root-health.md

```diff
@@ -38,3 +38,29 @@ backend.add(
   }),
 );
 ```
+
+### Custom headers in health check responses
+
+While not implemented directly in the root health service, the default implementation of the [RootHttpRouter](./root-http-router.md) service includes a configuration option to set additional headers to include in health check responses. For example, you can add a `service-name` header using the following configuration:
+
+```yaml
+backend:
+  health:
+    headers:
+      service-name: my-service
+```
+
+It can be a good idea to set a header for your health check responses that
```

Comment:
Hey 👋 

Just a QQ - _Should_ I be doing this, or I _can_ do this? 😅

Reply:
You *can*, it's by no means a requirement and really depends on what your infra looks like.